language/stdlib/modules/VASP.move

```diff
@@ -52,11 +52,13 @@ module VASP {
     public fun publish_parent_vasp_credential(vasp: &signer, lr_account: &signer) {
         assert(Roles::has_libra_root_role(lr_account), ENOT_LIBRA_ROOT);
         let vasp_addr = Signer::address_of(vasp);
-        assert(!is_vasp(vasp_addr), ENOT_A_VASP);
+        assert(Roles::has_parent_VASP_role(vasp), ENOT_A_PARENT_VASP);
```

Comment:
@shazqadeer This assertion is what I was mentioning this morning  that should allow us to prove "has parent vasp resource ==> has parent vasp role"